futures/tests_disabled/stream.rs

```diff
@@ -158,6 +158,12 @@ fn take_while() {
                 Ok(vec![1, 2]));
 }
 
+#[test]
+fn scan() {
```

Comment:
Could you move this test into `futures/tests/stream.rs` (`futures/tests_disabled/stream.rs` is not tested)